### PR TITLE
Added things to MakeFile

### PR DESCRIPTION
**Pull request by Jorropo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,2 +1,17 @@
+class:
+   maven compile -B -T 1C
```
Comment (juanj) [inline Makefile]: I believe Makefiles need to be indented with tabs

Comment (Jorropo) [reply Makefile]: Effectivly, atom just take the same indent as beffore

**Comment by theraccoonbear** [comment]
Might I suggest adding a `.editorconfig` to the project? http://editorconfig.org/

**Comment by Jorropo** [comment]
@theraccoonbear no, we don't need one cause .gitattribute manage end of line (it manage better than yours cause with .gitattibute it will auto convert LF to CRLF on windows when fetched and reconvert again CRLF to LF when pushed (and it does nothings on linux)). And for indentation we have google java format (juste execute format script for make an auto indentation).

PS : Suggestion is in issue normaly, not in a pull request :wink:

**Comment by theraccoonbear** [comment]
I suggested it here because it was being discussed here, but sure, it could be opened as an issue if someone felt it was worthwhile.  I've had very good luck with .editorconfig "just working" when collaborating on a team, but if everyone committing and sending PRs wants to manually run a formatter script then great.

**Pull request merged**